scripts/zones/Bostaunieux_Oubliette/IDs.lua

```diff
@@ -38,37 +38,14 @@ zones[xi.zone.BOSTAUNIEUX_OUBLIETTE] =
     },
     mob =
     {
-        SEWER_SYRUP_PH          =
-        {
-            [17461305] = 17461307, -- -19.000 1.000 -173.000
-            [17461306] = 17461307, -- -20.000 1.000 -148.000
-        },
-        SHII_PH                 =
-        {
-            [17461311] = 17461315, -- -59.000 0.941 -149.000
-            [17461334] = 17461315, -- -64.000 -0.500 -144.000
-            [17461277] = 17461315, -- -65.000 -1.000 -137.000
-            [17461309] = 17461315, -- -64.000 0.950 -132.000
-            [17461312] = 17461315, -- -53.000 -0.500 -137.000
-            [17461308] = 17461315, -- -57.000 0.998 -135.000
-        },
-        ARIOCH_PH               =
-        {
-            [17461322] = 17461433, -- -259 0.489 -188
-        },
-        MANES_PH                =
-        {
-            [17461469] = 17461471,
-            [17461470] = 17461471,
-            [17461476] = 17461471,
-            [17461477] = 17461471,
-        },
-        DREXERION_THE_CONDEMNED = 17461338,
-        PHANDURON_THE_CONDEMNED = 17461343,
+        SEWER_SYRUP             = GetFirstID('Sewer_Syrup'),
+        ARIOCH                  = GetFirstID('Arioch'),
+        DREXERION_THE_CONDEMNED = GetFirstID('Drexerion_the_Condemned'),
+        PHANDURON_THE_CONDEMNED = GetFirstID('Phanduron_the_Condemned'),
         BLOODSUCKER             = 17461478,
```

Comment:
Would something like `GetTableOfIDs('Bloodsucker')[X],` be better?

Reply:
Ah thank you, I meant to put it as a question completely skipped my mind. I wasn’t sure if we wanted a tableid[] or not because there are 15 bloodsuckers I think (on phone cant look).I was brainstorming to see if I could come up with something else that made more sense but I think thats the best idea.

Reply:
Updated to [42]. I wish we had an easier way of doing this, it just looks/feels really odd

Reply:
Would be nice if we had something like GetFirstNMId() but I don't think there's any way to differentiate the records at the moment.

Reply:
Nope, its just one dumb list (well, two dumb lists that should be one, but that's a story for another day between NPCs and Mobs)